test/test-threadpool-cancel.c

```diff
@@ -77,11 +72,23 @@ static void saturate_threadpool(void) {
 static void unblock_threadpool(void) {
   size_t i;
 
-  for (i = 0; i < ARRAY_SIZE(pause_reqs); i += 1)
+  for (i = 0; i < start_size; i += 1)
     uv_sem_post(pause_sems + i);
 }
 
 
+void threadpool_saturate(unsigned start) {
+  start_size = start;
+  ASSERT(start <= ARRAY_SIZE(pause_reqs));
+  saturate_threadpool();
+}
+
+
+void threadpool_unblock(void) {
+  unblock_threadpool();
+}
```

Comment:
Why these wrapper functions?  Why not expose them directly?

Reply:
The underlying function is called a dozen times through the file, I was looking to avoid churn where the existing functions were already in use. I renamed them: b7761de